signup.css

```diff
@@ -0,0 +1,107 @@
+* {
+  margin:0;
+  padding:0;
+  box-sizing: border-box;
+}
+body {
+  background-color:#f7f4f4;
+  font-family:sans-serif;
+}
+.blue {
+  color:#1F59BB;
+  text-transform: capitalize;
+  font-weight: 400;
+}
+.info {
+  font-size: 13px;
+  font-weight: 600;
+  text-transform: capitalize;
+}
+.signUp {
+  padding:0rem 0.2rem;
+}
+.privacy {
+  font-weight: 300;
+  text-align: center;
+  margin-top: 1rem;
+  font-size: 12px;
+}
+.submit {
+  text-transform: capitalize;
+  text-align: center;
+  color:#fff;
+  width: 100%;
+  background-color: #1F59BB;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#1F59BB" to be "#1f59bb" (color-hex-case)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5576441)